PolyEngine/Core/Src/String.hpp

```diff
@@ -6,21 +6,13 @@
 namespace Poly {
 
 	template<typename T>
-	size_t StrLen(const T* str) {
-		size_t len = 0;
-		while (str[len] != 0)
-			++len;
-		return len;
-	}
-
-	template<typename T>
 	class StringBase : BaseObject<> {
 	public:
 		StringBase() { Data.Resize(1); Data[0] = 0; }
 
 		//------------------------------------------------------------------------------
 		StringBase(const T* data) {
-			size_t length = StrLen(data);
+			size_t length = std::strlen(data);
```

Comment:
we for sure need unicode, and probably utf-8 is best, but utf-16 is still very popular and may be neede to support files not created by our engine but by some outside editor.